eblob-bf.spec

```diff
@@ -64,6 +64,18 @@ appending records one after another.
 This package contains libraries, header files and developer documentation
 needed for developing software which uses the eblob library.
 
+%package python
```

Comment:
You want to force all users to change their dependencies, to install multiple packages, to monitor multiple packages and versions during updates, to try not to break update with invalid version, and all users will still install all packages, not one, but 3 anyway.

All of this just for 'I think this is right'?

This is convenient to have as little as possible infrastructure packages - infrastructure is designed to be self-hosting. Eblob is such a package. When one needs a low-level storage (like elliptics does) admin should not think about what has to be installed among 3 packages, there is just a single button to press.

Please do not break this convenience without major need.